docs/sidebars.js

```diff
@@ -59,7 +59,8 @@ module.exports = {
             'maps/adding-points-and-lines',
             'maps/latlon',
             'maps/map-series',
-            'maps/tilemap-series'
+            'maps/tilemap-series',
+            'maps/mapline-series'
```

Comment:
It would be nice to have the map series (other than the main one) sorted from a to z.